package.json

```diff
@@ -99,6 +99,9 @@
         "typeorm": "0.2.18",
         "typescript": "~4.3.2",
         "unidiff": "1.0.1",
+        "webpack-cli": "4.8.0",
```

Comment:
I agree. We need to find a way to avoid adding dependencies of packages into the root package. Since I need this to run the generated_file_test, I can’t really remove it.

Reply:
I think the point is moot considering all packages will be split out into their own repos in the future. If that changes we can address the problem, but in all likelyhood the packages will be removed as packages leave the monorepo